src/ArrayUtils.js

```diff
@@ -174,39 +174,40 @@ function applyDotProduct(firstVector, secondVector) {
  * @param options
  * @returns {*}
  */
-function scale(input, options){
-    var y;
-    if(options.inPlace){
-        y = input;
-    }
-    else{
-        y = new Array(input.length);
-    }
-    const max = options.max;
-    const min = options.min;
-    if(typeof max === "number"){
-        if(typeof min === "number"){
-            var minMax = Stat.minMax(input);
-            var factor = (max - min)/(minMax.max-minMax.min);
-            for(var i=0;i< y.length;i++){
-                y[i]=(input[i]-minMax.min)*factor+min;
+function scale(input, options = {}) {
+    const {
```

Comment:
There's no need to add them as `null`, if you don't assign any value it will be `undefined` by default